server/leader.go

```diff
@@ -90,9 +92,13 @@ func (s *Server) leaderLoop() {
 	}
 }
 
+func getLeaderAddr(leader *pdpb.Member) string {
+	return strings.Join(leader.GetClientUrls(), ",")
+}
+
 // getLeader gets server leader from etcd.
-func getLeader(c *clientv3.Client, leaderPath string) (*pdpb.Leader, error) {
-	leader := &pdpb.Leader{}
+func getLeader(c *clientv3.Client, leaderPath string) (*pdpb.Member, error) {
+	leader := &pdpb.Member{}
 	ok, err := getProtoMsg(c, leaderPath, leader)
```

Comment:
seem that it is not compatible with existing leader data in etcd. 
We must wait the old data expired and removed then it can work.